game_eggs/steamcmd_servers/satisfactory/egg-satisfactory.json

```diff
@@ -9,19 +9,20 @@
     "description": "Satisfactory is a first-person open-world factory building game with a dash of exploration and combat. Play alone or with friends, explore an alien planet, create multi-story factories, and enter conveyor belt heaven!",
     "features": null,
     "images": [
+        "ghcr.io\/parkervcp\/games:source",
```

Comment:
Why two different source images?

Reply:
Freedom of choice ¯\_(ツ)_/¯

Reply:
Sometimes a choice isn't necessary, since they both should be doing the same thing;  pushing different changes to one image might cause different outcomes

Reply:
Okay, I'll remove the Pterodactyl image since it tends to be out of date anyways.